frontend/app/movies/[id]/personList.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link";
+import Image from "next/image";
+
+type personListItem = {
+  id: number;
+  name: string;
+  profile_path: string | null;
+  character?: string;
+  job?: string;
+};
+
+interface personListProps {
+  persons: personListItem[];
+}
+
+const combinePersonData = (persons: personListItem[]): personListItem[] => {
```

Comment:
Mikä tän funktion tarkoitus on?

Reply:
Funktion paluu arvo tyypitys ei ole tarpeellinen

Reply:
Näyttelijät näyttäis olevan tmdb:ssä muodossa `character 1 / character 2 / ...`.
https://www.themoviedb.org/movie/9327-the-nutty-professor

Reply:
Joo, tästä tämän voi varmaan poistaa. Crewssä oli enemmän sitä että esim. ohjaaja ja käsikirjoittaja on sama henkilö.

Reply:
Esim The Shawshank Redemptionissa (278) oli ohjaaja ja käsikirjoittaja sama sekä Stephen Kingillä jobbina Thanks ja Novel.

Reply:
Hmm tosin nyt huomasin että just The Shawshank Redemptionissa kun teen tuon kuuden henkilön slicen, se yhdistelee ne ja sivulla näkyykin vaan neljä tyyppiä. Pitää vielä vähän ihmetellä tätä.

Reply:
slice varmaan yhdistelyn jälkeen